src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 3 && days < 6) {
+    return 40 * days - 20;
+  }
+
+  if (days === 6) {
+    return 40 * days - 20;
+  }
```

Comment:
This condition is unnecessary. You're checking if the number of days is equal to 6 and then applying the same calculation as in the previous condition. This can be included in the previous condition by simply checking if the number of days is less than 7 instead of 6.